lib/gateway/Shard.js

```diff
@@ -560,8 +538,6 @@ class Shard extends EventEmitter {
     sendStatusUpdate() {
         this.sendWS(GatewayOPCodes.PRESENCE_UPDATE, {
             activities: this.presence.activities,
```

Comment:
```suggestion
            activities: this.presence.activities,
            afk: !!this.presence.afk, // For push notifications
            since: this.presence.status === "idle" ? Date.now() : 0,
```

Presence updates don't work without these lines

Reply:
`since` is optional.

Reply:
it doesn't work without it, so no it isn't

Reply:
since is *nullable*
![image](https://user-images.githubusercontent.com/17226394/167316545-f98a7ae4-da29-4e48-9467-20cac4cfa613.png)